control-plane/api-gateway/common/resources.go

```diff
@@ -605,6 +633,34 @@ func (s *ResourceMap) HTTPRouteCanReferenceBackend(route gwv1beta1.HTTPRoute, re
 	return s.referenceValidator.HTTPRouteCanReferenceBackend(route, ref)
 }
 
+func (s *ResourceMap) getConsulKey(route *gwv1beta1.HTTPRoute) api.ResourceReference {
```

Comment:
Be careful with this method, I intentionally kept some of the normalization code a bit heavy in this implementation due to the care that needs to be taken around when we do/don't normalize our lookups based on how non-enterprise Consul freaks out if you pass any sort of Namespace/Partition data back. I think it should be fine in your usage below since you're not trying to sync anything back to Consul and inlining the filters to the route resource, but the easier we make some of this normalized key construction, the easier it is to accidentally break non-enterprise builds.

Reply:
Went ahead and just removed it since I'm not longer using this function.